addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -288,6 +298,24 @@ export class Rtc extends Record {
             }
             this.call();
         }, 30_000);
+
+        this.multiTab.bus.addEventListener("discuss.rtc/recover", async ({ detail }) => {
+            if ((new Date().getTime() - detail.timestamp) > RECOVERY_DELAY) {
+                return;
+            }
+            setTimeout(async () => {
+                if (!this.multiTab.isOnMainTab()) {
+                    return;
+                }
+                const channel = this.store.Thread.get({
+                    model: "discuss.channel",
+                    id: detail.channelId,
+                });
+                if (channel && !this.state.channel && Object.keys(channel.rtcSessions).length > 1) {
```

Comment:
`getOrFetch()` looks more appropriate than `get()`, so that we can recover call even if we don't have local knowledge on main tab.